sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/RecognizeCustomFormsLiveTests.cs

```diff
@@ -410,28 +410,18 @@ public async Task StartRecognizeCustomFormsWithoutLabels(bool useStream, bool in
         }
 
         [RecordedTest]
-        [TestCase(true)]
-        [TestCase(false, Ignore = "https://github.com/Azure/azure-sdk-for-net/issues/12319")]
-        public async Task StartRecognizeCustomFormsWithoutLabelsCanParseMultipageForm(bool useStream)
+        public async Task StartRecognizeCustomFormsWithoutLabelsCanParseMultipageForm()
```

Comment:
The service hasn't fixed this issue in years, so we're splitting the cases in two separate tests: one that works normally and another one that expects an exception.